etc/neutron/plugins/cisco/aci_asr_config.ini

```diff
@@ -0,0 +1,8 @@
+{
+    'Datacenter-Out': {
+        'gateway_ip': '1.103.2.1',
```

Comment:
Just checked with mandeep, he also wants consistency across both sides otherwise there will be confusion to the customers....So what I proposed above is the consistent way to do it. We can file a separate bug if you don't like the existing naming.